src/main/java/org/radarcns/management/service/SubjectService.java

```diff
@@ -126,7 +126,7 @@ private Role getProjectParticipantRole(ProjectDTO projectDTO) {
 
 
     @Transactional
-    public SubjectDTO updateSubject(SubjectDTO subjectDTO) throws IllegalAccessException {
+    public SubjectDTO updateSubject(SubjectDTO subjectDTO) {
```

Comment:
any idea why we had this exception earlier?

Reply:
Going back in the history, it was added in [this commit](https://github.com/RADAR-CNS/ManagementPortal/commit/df0608278860f9f0b27ae94cdae3386553cdf87a#diff-359c75b052347bcf9e55b4901495674eR109), but no other changes were made to the method in that commit. So I assume it was added when working on the method, and that the throwing code got removed again before the commit. In any case i don't see anything there now that would throw an `IllegalAccessException`.